lucene/sandbox/src/test/org/apache/lucene/search/TestIndexSortSortedNumericDocValuesRangeQuery.java

```diff
@@ -65,7 +65,7 @@ public void testSameHitsAsPointRangeQuery() throws IOException {
         iw.deleteDocuments(LongPoint.newRangeQuery("idx", 0L, 10L));
       }
       final IndexReader reader = iw.getReader();
-      final IndexSearcher searcher = newSearcher(reader, false);
+      final IndexSearcher searcher = newSearcher(reader);
```

Comment:
This isn't critical for test coverage, but it seemed off that we had disabled wrapping the reader.